### PR TITLE
replace project-template with opensearch

### PR DESCRIPTION
**Pull request by sdelamo**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,10 +1,10 @@
 projectVersion=1.0.0-SNAPSHOT
-projectGroup=io.micronaut.project-template
+projectGroup=io.micronaut.opensearch
 
-title=Micronaut project-template
+title=Micronaut opensearch
 projectDesc=TODO
```
Comment (sdelamo) [inline gradle.properties:5]:
```suggestion
projectDesc=Micronaut integration with OpenSearch
```

File: MAINTAINING.md [hunk 2]
```diff
@@ -115,7 +115,7 @@ Renovate and our workflow will keep sending PRs. You should also create an issue
 
 #### Files sync
 
-We have a [template repo](https://github.com/micronaut-projects/micronaut-project-template) that we use as the single
+We have a [template repo](https://github.com/micronaut-projects/micronaut-opensearch) that we use as the single
```
Comment (wetted) [inline MAINTAINING.md:118]: I think this is an erroneous change.

**Review by wetted — CHANGES_REQUESTED** [review]

File: MAINTAINING.md (hunk 2 above)
Comment (sdelamo) [inline MAINTAINING.md:118]:
```suggestion
We have a [template repo](https://github.com/micronaut-projects/micronaut-project-template) that we use as the single
```

Comment (sdelamo) [reply MAINTAINING.md]: resolved via https://github.com/micronaut-projects/micronaut-opensearch/pull/2/commits/f939149fff2bca038393253fc7626fd7ee33e2f5

**Review by wetted — APPROVED** [review]

**Pull request merged**